pkg/liqonet/tunnel/wireguard/driver.go

```diff
@@ -518,6 +546,17 @@ func (w *Wireguard) Collect(ch chan<- prometheus.Metric) {
 		return
 	}
 
+	var wgUsrActive float64
+	if w.Implementation == Userspace {
+		wgUsrActive = 1
+	}
+	ch <- prometheus.MustNewConstMetric(
+		WgUserImpl,
+		prometheus.GaugeValue,
+		wgUsrActive,
+		[]string{liqoconst.DriverName}...,
+	)
```

Comment:
I would use the metrics differently. We can define a GaugeVect and then use a different label value for each implementation type (similar to how it is done with k8s pod phases). In this way is clearer to read the metric, and we can also support multiple values for that metric

![Screenshot 2023-02-09 at 09 21 56](https://user-images.githubusercontent.com/36957454/217756473-b0d0430a-1db5-441d-af0c-39bad8e857f1.png)